distribution/scripts/jmeter/create-summary-markdown.py

```diff
@@ -68,7 +68,7 @@ def main():
         }
 
     markdown_filename = "summary.md"
-    with open(markdown_filename, 'w') as f:
+    with open(markdown_filename, 'wb') as f:
```

Comment:
Why do we need this?

Reply:
Following error is logged without this change.
```
Traceback (most recent call last):
  File "jmeter/create-summary-markdown.py", line 77, in <module>
    main()
  File "jmeter/create-summary-markdown.py", line 73, in main
    f.write(markdown_file.encode("utf-8"))
TypeError: write() argument must be str, not bytes
```
This was a suggested answer in [here](https://stackoverflow.com/questions/5512811/builtins-typeerror-must-be-str-not-bytes).

Reply:
Okay. Thanks for the clarification.